src/firebase/firestore.js

```diff
@@ -0,0 +1,5 @@
+import { getFirestore } from "firebase/firestore";
+
+import app from "./app";
+
+export const db = getFirestore(app);
```

Comment:
Sets up the connection to the firestore db, and gives us 'db' as a const to be imported and used elsewhere